CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [#12187](https://github.com/cosmos/cosmos-sdk/pull/12187) Add batch operation for x/nft module.
 * [#12453](https://github.com/cosmos/cosmos-sdk/pull/12453) Add `NewInMemoryWithKeyring` function which allows the creation of in memory `keystore` instances with a specified set of existing items.
 * [#11390](https://github.com/cosmos/cosmos-sdk/pull/11390) `LatestBlockResponse` & `BlockByHeightResponse` types' `Block` filed has been deprecated and they now contains new field `sdk_block` with `proposer_address` as `string`
+* [#12576](https://github.com/cosmos/cosmos-sdk/pull/12576) Remove dependency on cosmos/keyring and upgrade to 99designs/keyring v1.2.1
```

Comment:
Can we have this on top?

Reply:
Sure! Done in https://github.com/cosmos/cosmos-sdk/pull/12576/commits/9c9f25fe226bcd48436dc8f5b323b9bab32a9588. For my edification, what order are these changelog entries sorted in?

Reply:
The latest entry should simply be on top, although occasionally, we forget, which leaves some kind of funky sorting.